src/Common/FitOnFhir.Common/Models/AuthorizeResponseData.cs

```diff
@@ -0,0 +1,30 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.Health.FitOnFhir.Common.Serialization;
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.FitOnFhir.Common.Models
+{
+    public class AuthorizeResponseData
+    {
+        [JsonConstructor]
+        public AuthorizeResponseData()
+        {
+        }
+
+        public AuthorizeResponseData(string authUrl, DateTimeOffset expiresAt)
+        {
+            AuthUrl = authUrl;
+            ExpiresAt = expiresAt;
+        }
+
+        [JsonRequired]
+        public string AuthUrl { get; set; }
```

Comment:
For objects that get serialized and deserialized, it's a good idea to add the JsonPropertyAttribute to ensure that the string representation of the property is explicitly defined.

e.g. `[JsonProperty("authUrl")]`